zds/utils/templatetags/profile.py

```diff
@@ -12,47 +12,47 @@
 
 
 @register.filter('profile')
-def profile(user):
+def profile(the_user):
     try:
-        profile = user.profile
+        the_profile = the_user.profile
     except Profile.DoesNotExist:
-        profile = None
-    return profile
+        the_profile = None
+    return the_profile
 
 
 @register.filter('user')
-def user(pk):
+def user(the_pk):
     try:
-        user = User.objects.get(pk=pk)
-    except:
-        user = None
-    return user
+        the_user = User.objects.get(pk=the_pk)
+    except User.DoesNotExist:
+        the_user = None
+    return the_user
 
 
 @register.filter('state')
-def state(user):
+def state(the_user):
     try:
-        profile = user.profile
-        if not profile.user.is_active:
-            state = 'DOWN'
-        elif not profile.can_read_now():
-            state = 'BAN'
-        elif not profile.can_write_now():
-            state = 'LS'
-        elif user.has_perm('forum.change_post'):
-            state = 'STAFF'
+        the_profile = the_user.profile
+        if not the_profile.user.is_active:
+            the_state = 'DOWN'
+        elif not the_profile.can_read_now():
+            the_state = 'BAN'
+        elif not the_profile.can_write_now():
+            the_state = 'LS'
+        elif the_user.has_perm('forum.change_post'):
+            the_state = 'STAFF'
         else:
-            state = None
+            the_state = None
     except Profile.DoesNotExist:
-        state = None
-    return state
+        the_state = None
+    return the_state
```

Comment:
oui mais sans les "the" on a des noms de variables qui ont les memes noms que les methodes, et ca c'est pas bo !